src/software/thunderscope/robot_communication.py

```diff
@@ -188,8 +188,8 @@ def __enter__(self):
         # TODO (#2741): we might not want to support robot diagnostics in tscope
         # make a ticket here to create a widget to call these functions to detach
         # from AI and connect to robots/or remove
-        # self.disconnect_fullsystem_from_robots()
-        # self.connect_robot_to_diagnostics(0)
+        self.disconnect_fullsystem_from_robots()
+        self.connect_robot_to_diagnostics(0)
```

Comment:
I think this should remain as it was (commented out), and #2789 will take care of it.

Reply:
done